rental.rb

```diff
@@ -7,8 +7,10 @@ class Rental
 
   def initialize(date, book, person)
     @date = date
+
     @book = book
     book.rental << self
+
     @person = person
     person.rental << self
   end
```

Comment:
- [ ] When the user tries to list all rentals for a given ID, the following error occurs and the program is terminated, kindly fix this.
![image](https://user-images.githubusercontent.com/81932478/217520307-f3a5e0b4-6ace-41e1-ae0a-5b77b1385c57.png)